util/slice/slice.go

```diff
@@ -0,0 +1,28 @@
+package slice
+
+func RemoveString(slice []string, element string) []string {
+	n := len(slice)
+	if n == 1 {
+		return []string{}
+	}
```

Comment:
This returns an empty string without even looking at `element`? What if `slice[0] != element`?